macos/FluentUITestViewControllers/TestLabelViewController.swift

```diff
@@ -0,0 +1,87 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import AppKit
+import FluentUI
+
+
+class TestLabelViewController: NSViewController {
+
+	override func loadView() {
+
+
+		let labels = TextStyle.allCases.map {textStyle -> NSTextField in
+			return Label(withString: string(ForTextStyle: textStyle), forTextStyle: textStyle)
+		}
+		
+		let buttons = TextStyle.allCases.map {textStyle -> Button in
+			return Button(title: string(ForTextStyle: textStyle), textStyle: textStyle, style: .primaryFilled)
```

Comment:
i just feel that it is odd that we show button and link in label test code

Reply:
Yeah, it was more of a "TextStyle" test view controller, than a label view controller...